Makefile

```diff
@@ -145,7 +145,7 @@ e2e-fpga:
 	@$(GO) test -v ./test/e2e/... -ginkgo.v -ginkgo.progress -ginkgo.focus "FPGA" -delete-namespace-on-failure=false
 
 e2e-qat:
-	@$(GO) test -v ./test/e2e/... -ginkgo.v -ginkgo.progress -ginkgo.focus "QAT plugin in DPDK mode" -delete-namespace-on-failure=false
+	@$(GO) test -v ./test/e2e/... -ginkgo.v -ginkgo.progress -ginkgo.focus "QAT Gen4" -delete-namespace-on-failure=false
```

Comment:
Are we going to stop running DPDK test? Why? It worked just fine so far. I think we should keep running it.

Reply:
I'm expecting `QAT Gen4` to be it's own `e2e-` target. The existing `e2e-qat` should not change.

Reply:
So we can follow https://github.com/intel/intel-device-plugins-for-kubernetes/blob/main/.github/workflows/e2e-dsa.yml and make it e2e-qat4.yml, `name: e2e-qat4` which `runs-on: [self-hosted, linux, x64, dsa]` with the set of images needed for QAT + `openssl-qat-engine`.

Reply:
Do we want to run this test on a real hw machines as well?

Reply:
no, `e2e-qat` is not Gen4 and the tests expect that

Reply:
thanks. now it's clear for me.